strr-web/lang/en.json

```diff
@@ -124,7 +124,7 @@
     "principalResidenceReason": "Reason",
     "principalResidenceServiceProvider": "Service Provider",
     "principalResidenceNotApplies": "Principal residence does not apply or my property is exempt",
-    "principalResidenceApplies": "Pincipal residence does apply",
+    "principalResidenceApplies": "Principal residence does apply",
```

Comment:
That was reported as a bug, I move this one to Ready for QA.
https://app.zenhub.com/workspaces/strr-65b2a7146835aa0cdf315b79/issues/gh/bcgov/entity/23633

Thanks